python/python/glide/async_commands/redis_modules/json/commands.py

```diff
@@ -0,0 +1,109 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+import json
+from typing import Dict, List, Optional, Union, cast
+
+from glide.async_commands.core import ConditionalChange
+from glide.constants import TOK
+from glide.protobuf.redis_request_pb2 import RequestType
+from glide.redis_client import TRedisClient
+
+TJson = Union[str, int, float, bool, None, List["TJson"], Dict[str, "TJson"]]
+
+
+class Json:
+    """Class for RedisJson commands."""
+
+    async def set(
+        self,
+        client: TRedisClient,
+        key: str,
+        path: str,
+        value: TJson,
```

Comment:
true, but I think it could be nice to be able to pass a dictionary (or another complex object) and also to be able to pass a string representation of the dictionary.

Reply:
I agree with Shachar, if we expect that the user will use python's json module to convert the response from a string to a complex object, it's relevant to the passed value too. If we except only string as a value, we can avoid importing 'json' module, which might not be in the standard libraries in other languages.